JDI.Light/JDI.Light.Tests/Tests/TestBase.cs

```diff
@@ -20,8 +20,8 @@ public virtual void SetUpTest()
         {
             Jdi.Logger.Info("Test Base Set up started...");
             TestSite = Jdi.InitSite<TestSite>();
-            //TestSite.HomePage.Open();
-            HomePageStatic.Open();
+            TestSite.HomePage.Open();
+            //HomePageStatic.Open();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5599881)